### PR TITLE
(DOCSP-38401) Documents config file for projects update

### PR DESCRIPTION
**Pull request by sarahsimpers**
- [DOCSP-number](https://jira.mongodb.org/browse/DOCSP-38401)
- [STAGING](https://preview-mongodbsarahsimpers.gatsbyjs.io/atlas-cli/DOCSP-38401/reference/json/project-config-file/)
- [LATEST BUILD LOG](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=661fdb673a5920a29100b191)

### Self-Review Checklist

- [ ] Check that the submodule pulled in the right changes (if applicable). **N/A**
- [ ] [Define](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) taxonomy [values](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) at top of page.
- [ ] Add genre facets (tutorial or reference), as in this [example PR](https://github.com/10gen/cloud-docs/pull/5042).
- [ ] Add programmingLanguage (if necessary).
- [ ] Add meta keywords (if necessary).
- [ ] Resolve any new warnings or errors in the build.
- [ ] Proofread for spelling and grammatical errors.
- [ ] Check staging for rendering issues.
- [ ] Confirm links are working.

### CODE REVIEW

File: source/includes/steps-config-file-template.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+      a. Navigate to the |openapi-link|
+         section of the |service| Admin API specification.
+      b. Under :guilabel:`Request samples` on the right side, click
+         :guilabel:`Expand all`.
+      c. Click :guilabel:`Copy` to copy the sample request.
+
+   .. step:: Create the configuration file.
+    
+      a. Paste the copied sample request into a text editor and change
```
Comment (pierwill) [inline source/includes/steps-config-file-template.rst:16]: Just a suggestion.

```suggestion
      a. Paste the sample request into a text editor and change
```

File: source/includes/steps-config-file-template.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+         the values to reflect your values.
```
Comment (pierwill) [inline source/includes/steps-config-file-template.rst:17]:
```suggestion
         the values to reflect your desired configuration.
```

File: source/reference/json/project-config-file.txt [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+   :name: genre
+   :values: tutorial
+
+.. To create a configuration file doc, copy this template, change
+.. the replacements below, and update the H1 title and refs.
+
+.. |atlas-cli command| replace:: :ref:`atlas-projects-update`
+.. |configuration-file-name| replace:: project configuration  
+.. |openapi-link| replace:: :oas-atlas-tag:`Update One Project </Projects/operation/updateProject>`
+.. |action| replace:: update a project
```
Comment (pierwill) [inline source/reference/json/project-config-file.txt:23]: This text is commented out, yes?

**Review by pierwill — APPROVED** [review]
LGTM with a few suggestions.

**Review by tibulca — APPROVED** [review]

**Pull request merged**